<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>362</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>신규 회계구분 단위 등록</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>사용자 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>회계단위 구분</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_gubun_widget">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>83</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addgubun_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>82</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>신규등록 추가</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="registed_gubun_tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>141</width>
     <height>192</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <column>
    <property name="text">
     <string>기존 등록된 사항</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="new_gubun_tableWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>141</width>
     <height>192</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <column>
    <property name="text">
     <string>신규등록 </string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="gubunsave_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>232</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>저장(F12)</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gubuncancel_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>280</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gubun_view_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>다시 보기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selected_row_delete_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>선택 행 삭제</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>new_gubun_widget</tabstop>
  <tabstop>addgubun_button</tabstop>
  <tabstop>gubun_view_button</tabstop>
  <tabstop>gubunsave_button</tabstop>
  <tabstop>selected_row_delete_button</tabstop>
  <tabstop>gubuncancel_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>addgubun_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>gubun_reg_input()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gubunsave_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>file_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gubuncancel_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>file_save_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gubun_view_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>registed_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selected_row_delete_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>delete_selected_row()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>477</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>gubun_reg_input()</slot>
  <slot>addgubun_cancel()</slot>
  <slot>file_save()</slot>
  <slot>file_save_cancel()</slot>
  <slot>registed_data()</slot>
  <slot>delete_selected_row()</slot>
 </slots>
</ui>
